lib/foreman_rescue/engine.rb

```diff
@@ -54,7 +54,7 @@ class Engine < ::Rails::Engine
       Host::Managed.prepend ForemanRescue::HostExtensions
       HostsHelper.prepend ForemanRescue::HostsHelperExtensions
       Nic::Managed.prepend ForemanRescue::Orchestration::TFTP
-      ProvisioningTemplate.prepend ForemanRescue::ProvisioningTemplateExtensions
+      ProvisioningTemplate.singleton_class.prepend ForemanRescue::ProvisioningTemplateExtensions::ClassMethods
```

Comment:
The place I stole this idea from uses `.send :prepend` and I have no idea why :see_no_evil: